plugins/redgifs.js

```diff
@@ -25,7 +25,22 @@ hoverZoomPlugins.push({
                 }
             });
 
-            $.get(`${apiUrl}/gifs/${gfyId}`, function(data) {
+            chrome.runtime.sendMessage({
+                action: 'ajaxGet',
+                url: `${apiUrl}/gifs/${gfyId}`,
+                headers: [{
+                    header: 'Authorization',
+                    value: `Bearer ${tempToken}`
+                }]
+            }, (response) => {
+                var data;
+
+                try {
+                    data = JSON.parse(response);
+                } catch (e) {
+                    return;
+                }
```

Comment:
Why swallow exceptions? When this approach eventually breaks, it will be easier to debug if it logs to console.